data/gegno/gegno ships.txt

```diff
@@ -766,19 +768,19 @@ ship "Protolith"
 		category "Heavy Freighter"
 		licenses
 			"Vi Lord"
-		"cost" 17600000
+		"cost" 168820000
```

Comment:
Everything looks pretty good so far. The only question I have is this, did you put an extra 0 or something by accident? The Conglomerate, a somewhat similarly-sized vessel, is 5,909,000, while this is 168,820,000. The Bactrian is 17,600,000. Is this supposed to be 16,882,000?
```suggestion
		"cost" 16882000
```

Reply:
Per the spreadsheet, it was actually an accidental extra 8.

Reply:
Dang it ol' spreadsheet.